py3-scipy.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: py3-scipy
+  version: 1.11.2
+  epoch: 0
+  description: Fundamental algorithms for scientific computing in Python
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    runtime:
+      - numpy
```

Comment:
Does this need py3-pythran at runtime too?

Reply:
don't know much about the package but I used `melange convert python` to generate the configuration file, it didn't add that package to runtime deps

Reply:
That might just be a bug in melange though. We'll find out when we make an image out of this whether we need it as a runtime dep too.